docs/INSTALL.md

```diff
@@ -93,16 +93,16 @@ following build dependencies are required:
 
 ### Installing Go
 
-`lnd` is written in Go, with a minimum version of `1.22.6` (or, in case this
+`lnd` is written in Go, with a minimum version of `1.23.6` (or, in case this
 document gets out of date, whatever the Go version in the main `go.mod` file
 requires). To install, run one of the following commands for your OS:
 
 <details>
   <summary>Linux (x86-64)</summary>
   
   ```
-  wget https://dl.google.com/go/go1.22.6.linux-amd64.tar.gz
-  sha256sum go1.22.6.linux-amd64.tar.gz | awk -F " " '{ print $1 }'
+  wget https://dl.google.com/go/go1.23.6.linux-amd64.tar.gz
```

Comment:
Need to update the hash here. And there's an instance of Go `1.22.6` below that should be updated as well.